datenguide_python/query_builder.py

```diff
@@ -76,34 +130,32 @@ def _get_fields_to_query(self, field: Union[str, "Field"]) -> str:
 
             if field.fields:
                 substring += "{"
-                for subfield in field.fields:
-                    substring += field._get_fields_to_query(subfield)
+                for key, field_item in field.fields.items():
```

Comment:
`key` does not appear to be used
```suggestion
                for field_item in field.fields.values():
```